code/ecdsap256sha2/Makefile

```diff
@@ -32,11 +32,13 @@ dist/libecdsa.a: dist/Hacl_ECDSA.o
 dist/%.o: dist/Makefile.basic
 	$(MAKE) -C dist -f Makefile.basic $(notdir $@)
 
-dist/Hacl_Test_ECDSA.exe: dist/Hacl_Test_ECDSA.o \
+dist/Hacl_Test_ECDSA.exe: dist/Hacl_ECDSA.o dist/Hacl_Test_ECDSA.o \
```

Comment:
this will trigger two parallel invocations of a recursive make in dist which will race with each other -- I would suggest depending only on `dist/libecsda.a` and changing `dist/%.o` above to be `dist/libecsda.a`